translations/ja.json

```
@@ -500,56 +500,56 @@
-	"Run a kubectl binary matching the cluster version": "クラスタのバージョンに適合する kubectl のバイナリを実行します",
-	"Run minikube from the C: drive.": "",
-	"Run the Kubernetes client, download it if necessary. Remember -- after kubectl!\n\nThis will run the Kubernetes client (kubectl) with the same version as the cluster\n\nNormally it will download a binary matching the host operating system and architecture,\nbut optionally you can also run it directly on the control plane over the ssh connection.\nThis can be useful if you cannot run kubectl locally for some reason, like unsupported\nhost. Please be aware that when using --ssh all paths will apply to the remote machine.": "",
-	"Run: 'Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V-Tools-All'": "",
-	"Run: 'kubectl delete clusterrolebinding kubernetes-dashboard'": "",
-	"Run: 'minikube delete --all' to clean up all the abandoned networks.": "",
-	"Run: 'sudo chown $USER $HOME/.kube/config \u0026\u0026 chmod 600 $HOME/.kube/config'": "",
-	"Run: 'sudo mkdir /sys/fs/cgroup/systemd \u0026\u0026 sudo mount -t cgroup -o none,name=systemd cgroup /sys/fs/cgroup/systemd'": "",
-	"Running on localhost (CPUs={{.number_of_cpus}}, Memory={{.memory_size}}MB, Disk={{.disk_size}}MB) ...": "",
-	"Running remotely (CPUs={{.number_of_cpus}}, Memory={{.memory_size}}MB, Disk={{.disk_size}}MB) ...": "",
-	"SSH key (ssh driver only)": "",
-	"SSH port (ssh driver only)": "",
-	"SSH user (ssh driver only)": "",
-	"Save a image from minikube": "",
-	"Select a valid value for --dnsdomain": "",
-	"Send trace events. Options include: [gcp]": "",
-	"Service '{{.service}}' was not found in '{{.namespace}}' namespace.\nYou may select another namespace by using 'minikube service {{.service}} -n \u003cnamespace\u003e'. Or list out all the services using 'minikube service list'": "",
+	"Returns logs to debug a local Kubernetes cluster": "ローカルの Kubernetes クラスターをデバッグするためのログを返します",
+	"Returns the Kubernetes URL for a service in your local cluster. In the case of multiple URLs they will be printed one at a time.": "ローカルクラスター中のサービス用 Kubernetes URL を返します。複数 URL の場合、それらは一度に出力されます。",
+	"Returns the value of PROPERTY_NAME from the minikube config file.  Can be overwritten at runtime by flags or environmental variables.": "minikube 設定ファイル中の PROPERTY_NAME の値を返します。実行時にフラグか環境変数を用いて上書きできます。",
+	"Right-click the PowerShell icon and select Run as Administrator to open PowerShell in elevated mode.": "PowerShell を特権モードで開くために、PowerShell アイコンを右クリックし、管理者として実行を選択してください。",
+	"Run 'kubectl describe pod coredns -n kube-system' and check for a firewall or DNS conflict": "'kubectl describe pod coredns -n kube-system' を実行し、ファイアウォールか DNS 衝突を確認してください",
+	"Run 'minikube delete' to delete the stale VM, or and ensure that minikube is running as the same user you are issuing this command with": "ストールした VM を削除するため、'minikube delete' を実行するか、次のコマンドを実行した時と同じユーザーとして minikube を実行している事を確認してください",
```

I think `stale` means `古い`.
And `this command` will be better `このコマンド` then `次のコマンド`.
This message may be printed for the following commands:
- `minikube start`
- `minikube node start`
- `minikube node add`

```suggestion
	"Run 'minikube delete' to delete the stale VM, or and ensure that minikube is running as the same user you are issuing this command with": "古い VM を削除するため、'minikube delete' を実行するか、このコマンドを実行した時と同じユーザーで minikube を実行している事を確認してください",
```

---

I agree with suggested update above.

---

I also agree.
In addition, the following is a Dashboard's translation policy.
>「〜なし」「〜のため」「〜すること」もひらがなで記載します。「〜無し」「〜の為」「〜する事」などは避けます。

So, "minikube を実行している事を" -> "minikube を実行していることを".

And based on this policy, I think "～したとき" is better than "～した時".
"このコマンドを実行した時" -> "このコマンドを実行したとき"